registry/client_commands.go

```diff
@@ -230,7 +231,7 @@ func NamespaceDelete(endpoint string, prefix string) error {
 	// TODO: We might consider moving widely-useful functions like `GetRegistryIssuerURL`
 	//       to a more generic `pelican/utils` package so that they're easier to find
 	//       and more likely to be used.
-	issuerURL, err := director.GetNSIssuerURL(prefix)
+	issuerURL, err := server_utils.GetNSIssuerURL(prefix)
```

Comment:
It looks like you can remove the previous TODO. I think this function has drifted both in name and location.